gooddata.gemspec

```diff
@@ -61,4 +61,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'salesforce_bulk_query', '~> 0.2'
   s.add_dependency 'terminal-table', '~> 1.7'
   s.add_dependency 'thread_safe'
+  s.add_dependency 'backports'
```

Comment:
It will allow us to use the great features of newer Rubies instead of polluting the code-base with workarounds. The good thing about `backports` is that it is only used when needed (i.e. running an old Ruby).